lib/Doctrine/DBAL/Driver/OCI8/OCI8Statement.php

```diff
@@ -134,7 +139,15 @@ static public function convertPositionalToNamedPlaceholders($statement)
      */
     public function bindValue($param, $value, $type = null)
     {
-        return $this->bindParam($param, $value, $type, null);
+        if ($type === \PDO::PARAM_LOB || -99 === $type) {
```

Comment:
I added this number to differ CLOB from BLOB. It's hard coded in the test and has nothing to do with oci8. I want to move this as a constant, where `getBindingType` knows (when using oracle) that this difference should be handled. Still, `getBindingType` is not platform aware if I'm not mistaken?